### ISSUE TITLE
v2.4.9.2

### ISSUE THREAD

**Issue by xbdmHQ** [pablouser1/ProxiTok#152]
After updating to latest using docker

```

Fatal error: Uncaught Latte\RuntimeException: Unable to create file '/cache/templates-views-home.latte--526dac6ae8.php.lock'. fopen(/cache/templates-views-home.latte--526dac6ae8.php.lock): Failed to open stream: Permission denied in /var/www/html/vendor/latte/latte/src/Latte/Engine.php:259 Stack trace: #0 /var/www/html/vendor/latte/latte/src/Latte/Engine.php(226): Latte\Engine->acquireLock('/cache/template...', 2) #1 /var/www/html/vendor/latte/latte/src/Latte/Engine.php(127): Latte\Engine->loadTemplate('/var/www/html/a...') #2 /var/www/html/vendor/latte/latte/src/Latte/Engine.php(99): Latte\Engine->createTemplate('/var/www/html/a...', Array) #3 /var/www/html/app/Helpers/Wrappers.php(122): Latte\Engine->render('/var/www/html/a...', Object(App\Models\BaseTemplate)) #4 /var/www/html/routes.php(14): App\Helpers\Wrappers::latte('home', Object(App\Models\BaseTemplate)) #5 [internal function]: {closure}() #6 /var/www/html/vendor/bramus/router/src/Bramus/Router/Router.php(458): call_user_func_array(Object(Closure), Array) #7 /var/www/html/vendor/bramus/router/src/Bramus/Router/Router.php(440): Bramus\Router\Router->invoke(Object(Closure), Array) #8 /var/www/html/vendor/bramus/router/src/Bramus/Router/Router.php(287): Bramus\Router\Router->handle(Array, true) #9 /var/www/html/index.php(17): Bramus\Router\Router->run() #10 {main} thrown in /var/www/html/vendor/latte/latte/src/Latte/Engine.php on line 259
```

**Comment by xbdmHQ** [comment]
[Resolved] I had to re-clone and pull and then up now it works.

https://cringe.whateveritworks.org/

**Comment by NoPlagiarism** [comment]
Could it be connected to PHP 8.2 (4a59dab75c0094f3f4de7e4040f18372406e683c)?

I guess that's all was resolved in 629da5f357fe1530210e9674ce3274f005bac398 and 34623c8e430e2ed8404c0c81a9f1494393b331aa

**Comment by xbdmHQ** [comment]
> Could it be connected to PHP 8.2 ([4a59dab](https://github.com/pablouser1/ProxiTok/commit/4a59dab75c0094f3f4de7e4040f18372406e683c))?
> 
> I guess that's all was resolved in [629da5f](https://github.com/pablouser1/ProxiTok/commit/629da5f357fe1530210e9674ce3274f005bac398) and [34623c8](https://github.com/pablouser1/ProxiTok/commit/34623c8e430e2ed8404c0c81a9f1494393b331aa)

Yeah, issue is fixed, latest version added on my instance. The others is unmaintained.

**Comment by NoPlagiarism** [comment]
> `The others is unmaintained.`

All instances are up for now, but version bump is needed indeed
